src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -320,6 +313,33 @@ public void checkifDownloadDirectoryisEmpty() throws IOException {
         assertFalse(Hooks.getDownloadDirectory().list().length > 0, "directory is not empty");
     }
 
+    /**
+     * Assert filename ends with date and time stamp.
+     * @param dateTimeFormat expected Date and Time
+     * @param fileName expected File name
+     */
+    @Then("^the \"([^\"]*)\" file ends with \"([^\"]*)\"$")
+    public void surveillanceActivityFileEndsWithDateTimeFormat(final String fileName, final String dateTimeFormat) {
```

Comment:
Why does this method say "surveillance" in the name, when it looks like it's supposed to be generic?